### ISSUE TITLE
SEO Integration for Sawaliram public pages

### ISSUE THREAD

**Issue by abhiprojectz** [sawaliram/sawaliram#429]
Sawaliram website is not **SEO** Friendly that is, google is unable to understand what sawaliram actually is , perhaps the below image describes it:

![gggggg](https://user-images.githubusercontent.com/64596494/124380208-1d265f00-dcd9-11eb-85f6-6d9096a38b76.PNG)

In the above image the description is just the headings and upper text , which is not clear & is the main reason of the site content not appearing in searches.

Implementing some Seo makes it easy for google (crawlers) to crawl the site and index it to the category for this to happen, **SEO** integration is required. We can implement various meta tags such as `description` , `author` etc. It will help to show our answers , articles etc, when searched by user.

**Comment by jr-sawaliram** [comment]
Thank you @abhiprojectz for identifying this issue. @badrihippo @nitinpaultifr please give your comments. Thanks.

**Comment by badrihippo** [comment]
Yes, like @abhiprojectz said you can just add the appropriate `<meta>` tags to the public website's `index.html` (or is it called `home.html`? Wherever that is located, basically). @abhiprojectz how about you give the list of available `<meta>` tags, and then @jr-sawaliram can suggest what to put in them?

If this works, we could think of adding tags for the other pages like articles, questions, etc. as well. Those will have to extract data from the questions/articles themselves somehow and would need a bit more coding work.

**Comment by abhiprojectz** [comment]
Okay @badrihippo   , i'll create a doc and link it later here, & add appropriate meta tags that will be best as per our project requirements.

and as you told @jr-sawaliram  may put the required data to be used for its content. There would be less pre-filled tags as on answers, articles the content will be fetched from models/Database.

**Comment by badrihippo** [comment]
Yes, start with putting meta tags on the homepage, and if that works then for the next round you can decide how to handle the generated meta tags for articles, answers, etc.

**Comment by abhiprojectz** [comment]
@jr-sawaliram  the [meta tags doc](https://docs.google.com/document/d/1b25nt6kEZ_h1Oeeq8PFolPulOOAewSTyZsgX_zMQneg/edit?usp=sharing) , the tags for each page is there, you may edit it.

**Comment by badrihippo** [comment]
@jr-sawaliram if you want to have an idea of how the meta tags would look in practice, [this website](https://metatags.io/) may help for mockup previews in Google Search, Facebook, Twitter, etc.

@abhiprojectz by `Image [alt]` do you mean an alternate image to use, or the alt text of the given image?

**Comment by abhiprojectz** [comment]
> or the alt text of the given image?
yes
